### DISCUSSION TITLE
Link component not navigating t0othe href specified in the app directory

### DISCUSSION THREAD

**Discussion by NathanielGunat** [vercel/next.js#41960] [Help]
The link  tag is not working in dev mode showing error:TypeError: Cannot read properties of undefined (reading '')
![Screenshot (2)](https://user-images.githubusercontent.com/115327192/198261119-877818e0-092c-4ac5-95fd-fd23d30c5fbd.png)

**Comment by icyJoseph** [comment]
Hi,

Could you share some code of how you are doing this? A repository perhaps? I am able to navigate between pages and app, and within app as well. Perhaps you have some rewrites/redirects, or middleware at play?

**Reply by icyJoseph** [reply]
I think using Turbopack (in windows?) causes this issue. Could you confirm?

**Reply by NathanielGunat** [reply]
I'm using windows 11. and I tried using the Nextjs 13 playgrounds template on vercel and it was still broken on my computer. this is how I implemented it 
[link to repository ](https://github.com/NathanielGunat/next-13)
